### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by fangyi-zhou**

### CODE REVIEW

**Comment by fangyi-zhou** [comment]
add linting javascript to CI

File: client/meter/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "error",
+            2
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "single"
```
Comment (Norbo11) [inline client/meter/.eslintrc.json]: I actually prefer double, is this a JS convention?

Comment (fangyi-zhou) [reply client/meter/.eslintrc.json]: It's a choice

Comment (fangyi-zhou) [reply client/meter/.eslintrc.json]: The use of quotes are inconsistent in the codebase. The choice here is rather arbitrary

File: dapp/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        ],
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+        "no-unused-vars": "warn"
```
Comment (fangyi-zhou) [inline dapp/.eslintrc.json]: It seems that eslint has issues with unused vars, sometimes react components are imported and used but are reported as unused.

**Comment by levex** [comment]
Woah this eslint is one damn hard to please

**Comment by fangyi-zhou** [comment]
Can merge now.

**Pull request merged**